server/src/main/java/org/opensearch/index/search/stats/SearchStats.java

```diff
@@ -147,13 +158,23 @@ private Stats(StreamInput in) throws IOException {
                 pitTimeInMillis = in.readVLong();
                 pitCurrent = in.readVLong();
             }
+
+            if (in.getVersion().onOrAfter(Version.V_2_10_0)) {
```

Comment:
👍

Reply:
@jed326 could you please create a pull request to main with these version changes only (in & out), we'll merge them in lockstep, thank you.

Reply:
@reta I'm about to publish a PR for https://github.com/opensearch-project/OpenSearch/issues/9644, would it be okay if I got that change merged into main first and then backport both at the same time? That way we don't have to deal with bwc backporting multiple times

Reply:
> @reta I'm about to publish a PR for #9644, would it be okay if I got that change merged into main first and then backport both at the same time? That way we don't have to deal with bwc backporting multiple times

Sure, but we cannot merge this one than, cuz if we do, all BWC tests on `main` would start to fail.

Reply:
Yep makes sense, let me convert this PR back to draft and then i'll re-use the same branch when backporting both commits.